modules/installation-aws-permissions-iam-roles.adoc

```diff
@@ -53,10 +60,24 @@ The control plane and compute machines require the following IAM role permission
 * `kms:DescribeKey`
 ====
 
-.Required IAM role permissions for compute instance profiles
+.Default IAM role permissions for worker instance profiles
 [%collapsible]
 ====
-* `sts:AssumeRole`
 * `ec2:DescribeInstances`
 * `ec2:DescribeRegions`
 ====
+
+= Custom permissions for IAM Instance Profiles
+
+Depending on your cluster set of operators and their configuration, it may be necessary to grant additional permissions to the instance profiles. Instead of manually adding each permission, you may https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_generate-policy.html?icmpid=docs_iam_console[generate policies based on access activity].
+
+The overall process would be:
+
+. https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-getting-started.html[Enable CLoudTrail], so that the API calls are logged.
```

Comment:
```suggestion
. https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-getting-started.html[Enable AWS CloudTrail], so that the API calls are logged.
```